scripts/deploy-backend.sh

```diff
@@ -27,9 +30,10 @@ generate_jwt_cert()
 
 update_backend()
 {
-    bash ./scripts/set-backend-env-vars.sh "$CGHOSTNAME_BACKEND" "$CF_SPACE"
     cd tdrs-backend || exit
     if [ "$1" = "rolling" ] ; then
+          bash ./scripts/set-backend-env-vars.sh "$CGHOSTNAME_BACKEND" "$CF_SPACE"
```

Comment:
Can you elaborate on why this was moved back? I was seeing bad behavior with `rebuild` strategy deployments. We can address during tabletop if preferable.

Reply:
Addressed in tabletop, note that this can be reverted later potentially